src/plots/cartesian/index.js

```diff
@@ -322,7 +322,11 @@ exports.clean = function(newFullData, newFullLayout, oldFullData, oldFullLayout)
     if(oldFullLayout._hasOnlyLargeSploms && !newFullLayout._hasOnlyLargeSploms) {
         for(k in oldPlots) {
             plotinfo = oldPlots[k];
-            if(plotinfo.plotgroup) plotinfo.plotgroup.remove();
+            if(plotinfo.plotgroup) {
+                for(var idx = 0; idx < plotinfo.plotgroup.length; idx++) {
```

Comment:
When removing it is safer to use a reversed loop e.g. sometimes the length of the array and the indices may change within the loop.